ts/session/utils/MessageUtils.ts

```diff
@@ -0,0 +1,89 @@
+import { RawMessage } from '../types/RawMessage';
+import { ContentMessage } from '../messages/outgoing';
+import { EncryptionType } from '../types/EncryptionType';
+import * as crypto from 'crypto';
+
+function toRawMessage(device: PubKey, message: ContentMessage): RawMessage {
+  const ttl = message.ttl();
+  const timestamp = message.timestamp;
+  const plainTextBuffer = message.plainTextBuffer();
+
+  // Get EncryptionType depending on message type.
+  // let encryption: EncryptionType;
+
+  // switch (message.constructor.name) {
+  //   case MessageType.Chat:
+  //     encryption = EncryptionType.Signal;
+  //     break;
+  //   case MessageType.SessionReset:
+  //     encryption = EncryptionType
+  // }
+
+  // export enum EncryptionType {
+  //   Signal,
+  //   SessionReset,
+  //   MediumGroup,
+  // }
+
+  // tslint:disable-next-line: no-unnecessary-local-variable
+  const rawMessage: RawMessage = {
+    identifier: message.identifier,
+    plainTextBuffer,
+    timestamp,
+    device: device.key,
+    ttl,
+    encryption: EncryptionType.Signal,
+  };
+
+  return rawMessage;
+}
+
+export enum PubKeyType {
+  Primary = 'priamry',
+  Secondary = 'secondary',
+  Group = 'group',
+}
+
+export class PubKey {
+  private static readonly regex: string = '^0[0-9a-fA-F]{65}$';
```

Comment:
```suggestion
  private static readonly regex: string = '^05[0-9a-fA-F]{64}$';
```
Don't know if this will break in testnet 🤔

Reply:
Done. Changed name to generateFake